Filters/VirtualCameraFilter.def

```diff
@@ -1,7 +1,7 @@
-LIBRARY     Filters.dll
-EXPORTS
-            DllMain                 PRIVATE
-            DllGetClassObject       PRIVATE
-            DllCanUnloadNow         PRIVATE
-            DllRegisterServer       PRIVATE
-            DllUnregisterServer     PRIVATE
+LIBRARY     Filters.dll
```

Comment:
Jeśli używamy VirtualCameraFilters.dll jak powyżej to tutaj raczej też powinno być?